packages/e2e-tests/tests/playwright-05_hover-dom-previews.test.ts

```diff
@@ -0,0 +1,85 @@
+import { openViewerTab, startTest } from "../helpers";
+import {
+  getSelectedTestCase,
+  getTestCaseSteps,
+  getTestRows,
+  getTestSuitePanel,
+  openPlaywrightTestPanel,
+} from "../helpers/testsuites";
+import { waitFor } from "../helpers/utils";
+import test, { expect } from "../testFixtureCloneRecording";
+
+test.use({ exampleKey: "playwright/breakpoints-05" });
+
+test("playwright-05: Test DOM node previews on user action step hover", async ({
+  pageWithMeta: { page, recordingId },
+  exampleKey,
+}) => {
+  await startTest(page, recordingId);
+  await openViewerTab(page);
+
+  await openPlaywrightTestPanel(page);
+
+  const testPanel = getTestSuitePanel(page);
+
+  const isVisible = await testPanel.isVisible();
+  expect(isVisible).toBe(true);
```

Comment:
nit: `openPlaywrightTestPanel` helper already ensures the test suite panel is visible.